invenio_rdm_records/services/config.py

```diff
@@ -209,6 +220,19 @@ class RDMFileRecordServiceConfig(FileServiceConfig, ConfiguratorMixin):
         "RDM_PERMISSION_POLICY", default=RDMRecordPermissionPolicy
     )
 
+    file_links_item = {
+        **FileServiceConfig.file_links_item,
+        # FIXME: should we check if the file is IIIF compatible?
+        # FIXME: filename instead
```

Comment:
Question: Do all files get IIIF links and manifest?

Reply:
Right now, yes. I'd imagine I can make the links conditional based on file extension and https://github.com/inveniosoftware/flask-iiif/blob/master/flask_iiif/config.py#L86

Reply:
Probably I should add the same logic at the record level. If there are IIIF compatible files, generate the manifest otherwise, don't do it.
But this will be trickier since I don't have access to the files at the record level ...